core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -952,19 +644,11 @@ object ConfigCommand extends Logging {
       val hasEntityDefault = entityNames.exists(_.isEmpty)
 
       val numConnectOptions = (if (options.has(bootstrapServerOpt)) 1 else 0) +
-        (if (options.has(bootstrapControllerOpt)) 1 else 0) +
-        (if (options.has(zkConnectOpt)) 1 else 0)
+        (if (options.has(bootstrapControllerOpt)) 1 else 0)
       if (numConnectOptions == 0)
-        throw new IllegalArgumentException("One of the required --bootstrap-server, --boostrap-controller, or --zookeeper arguments must be specified")
+        throw new IllegalArgumentException("One of the required --bootstrap-server or --bootstrap-controller arguments must be specified")
```

Comment:
I think this line is not reachable anymore since we now throw on line 627 is both --bootstrap-server and --bootstrap-controller not are present